cg/meta/workflow/rnafusion.py

```diff
@@ -4,6 +4,7 @@
 from pathlib import Path
 from typing import Dict, List, Optional
 
+from cg.models.rnafusion.analysis import RnafusionAnalysis
```

Comment:
In a future pr i would like to bring this into cg and remove the cg_model dependency